handbook/engineering/web/frontend-platform/index.md

```diff
@@ -84,6 +84,15 @@ We plan and track our day-to-day work on our [Kanban board](https://github.com/o
 
 The team makes use of [tracking issues](../../tracking_issues.md) for tracking progress on the implementation of new features. The teammates should ensure that a tracking issue is created when starting work on features that are expected to take longer than a few days (or require multiple PRs) to deliver.
 
+#### Getting cross-team feedback on RFC
+
+- Create a [tracking issue](../../tracking_issues.md) on our [Kanban board](https://github.com/orgs/sourcegraph/projects/144).
```

Comment:
Is the tracking issue still important if we already have an issue for the RFC? Can we not follow this process with one issue?

Reply:
1. The tracking issue will include links to other action items which should appear on RFC approval. It will allow tracking the RFC implementation progress whereas the RFC issue should be moved to `done` once it's approved.
2. Right. We mention this case in the [tracking issues section](https://about.sourcegraph.com/handbook/engineering/web/frontend-platform#tracking-issues)👇

> The teammates should ensure that a tracking issue is created when starting work on features that are expected to take longer than a few days (or require multiple PRs) to deliver

Reply:
Hmm I've only just noticed that the tracking issues text has changed. I think previously it was like `expected to take longer than a single iteration`. I feel like a tracking issue should created after an RFC is approved in this case, as it'll likely be empty until then and there will be some cases where the RFC is rejected.

My main reason for this is just to make it as easy as possible for us to follow this process (so we actually do it 😁 )

Reply:
I like the idea of keeping it as simple as possible so that we follow it.

But as an aside, let's remember that it's a request for _comments_ and not request for _approval_. :-) But, the points still stands that we might not actually pursue the work after getting the comments.

Reply:
Moved the tracking issue to the last step and made clear that it's optional for small features.